test/src/test/java/org/corfudb/integration/Harness.java

```diff
@@ -0,0 +1,98 @@
+package org.corfudb.integration;
+
+import org.corfudb.integration.AbstractIT;
+import org.corfudb.integration.cluster.Harness.Action;
+import org.corfudb.integration.cluster.Harness.Node;
+import org.corfudb.runtime.BootstrapUtil;
+import org.corfudb.runtime.view.Layout;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.corfudb.AbstractCorfuTest.PARAMETERS;
+import static org.corfudb.integration.AbstractIT.getCorfuServerLogPath;
+
+/**
+ * Utilities to form and manipulate a corfu cluster.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+
+public class Harness {
+
+    String localAddress = "localhost";
+
+    private String getAddressForNode(int port) {
+        return "localhost:900" + port;
```

Comment:
nit:i would just keep it as localhost  and ports can be 9000, 9001, 9002. Just easier to read.

Reply:
done